boards/nucleo-f103/Makefile.features

```diff
@@ -6,9 +6,6 @@ FEATURES_PROVIDED += periph_spi
 FEATURES_PROVIDED += periph_timer
 FEATURES_PROVIDED += periph_uart
 
-# load the common Makefile.features for Nucleo boards
-include $(RIOTBOARD)/common/nucleo64/Makefile.features
```

Comment:
Why removing this ?

Reply:
because then this board will not have the Arduino feature anymore and won't be used for the Arduino hello world examples :) other boards (like the f410/f411) don't have this feature either - I guess it was a copy paste for the f103 anyway, but never clashed before.